app/src/androidTest/java/org/mozilla/fenix/syncintegration/SyncIntegrationTest.kt

```diff
@@ -58,6 +60,27 @@ class SyncIntegrationTest {
         bookmarkAfterSyncIsShown()
     }
 
+    @Test
+    fun checkAccountSettings() {
+        signInFxSync()
+        mDevice.waitNotNull(Until.findObjects(By.text("Settings")), TestAssetHelper.waitingTime)
+
+        goToAccountSettings()
+        // This function to be added to the robot once the status of checkboxes can be checked
+        // currently is not possible to select each one (History/Bookmark) and verify its status
+        // verifyCheckBoxesSelected()
+        // Then select/unselect each one and verify again that its status is correct
+        accountSettings {
+            verifyBookmarksCheckbox()
+            verifyHistoryCheckbox()
+            verifySignOutButton()
+            verifyDeviceName()
+        }.disconnectAccount {
+            sleep(TestAssetHelper.waitingTime)
```

Comment:
I think you mentioned that this is necessary for now, but its gonna be brittle. The app must be waiting on something before you can successfully verify the SettingsView for a sleep to be needed.  Do we by chance know what that is?  If not, I can help research how we might wait on that... I suppose it's a disconnect signal of some kind?

Reply:
I will try to remove the sleep and wait for an element in the next screen before the next action. Hopefully that will work 🤞